docs/dist/pre-commit.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+CODE_SNIFFER=$(which phpcs)
+if [[ -z "$CODE_SNIFFER" ]];then
+    if cat /etc/*-release|grep -q '^ID=\(debian\|ubuntu\)';then
+        sudo apt-get install php-codesniffer
+        CODE_SNIFFER=$(which phpcs)
+    fi
+    if [[ -z "$CODE_SNIFFER" ]];then
```

Comment:
I like automating :) But I personally prefer downloading the current version of the codesniffer anyway, will remove the debian/ubuntu specific clause.

Not sure what you mean with the comments.